src/dmd/mars.d

```diff
@@ -732,6 +732,8 @@ bool parseCommandlineAndConfig(size_t argc, const(char)** argv, ref Param params
 
     version (Posix)
         global.preprocess = &preprocess;
+    version (Win64)
```

Comment:
You don't have to restrict this to dmd being compiled as a 64-bit program, `version(Windows)` should be fine, too.

Reply:
I don't want it to try the DMC win32 yet.

Reply:
The version is not about the target tool chain, but the host compiler. The standard dmd build will not execute the preprocessor.